apps/common/views/accessibility.html

```diff
@@ -37,15 +36,14 @@ <h2 class="govuk-heading-m">Non-accessible content</h2>
     <h3 class="govuk-heading-m">{{#t}}accessibility.non-compliance-header{{/t}}</h3>
     <ul class="govuk-list govuk-list--bullet">
         <li>{{#t}}accessibility.non-compliance-li1{{/t}}</li>
-        <li>A layout table is present, which is intended for visual formatting rather than conveying tabular data. This could create navigation and reading difficulties for screen reader users, as the table may be incorrectly announced as a data table. This does not meet WCAG 2.2 Meaningful Sequence and Info and Relationships.</li>
     </ul>
     <h3 class="govuk-heading-m">{{#t}}accessibility.disproportionate-burden-header{{/t}}</h3>
     <p class="govuk-body">Not Applicable</p>
     <h3 class="govuk-heading-m">{{#t}}accessibility.outside-scope-header{{/t}}</h3>
     <p class="govuk-body">Not Applicable</p>
     <h2 class="govuk-heading-m">Preparation of this accessibility statement</h2>
-    <p class="govuk-body">This statement was prepared on 28 September 2020. It was last reviewed on 10 September 2024.</p>
-    <p class="govuk-body">This website was last tested on 20 June 2024. The test was carried out internally by the Home Office.</p>
+    <p class="govuk-body">This statement was prepared on 6 December 2024. It was last reviewed on 6 December 2024.</p>
```

Comment:
Hi @tobyschaefferHO just noticed the dates are different on the on the accessibility statement word doc attached on the tickets. Are these the correct dates?

Reply:
Dates have been updated - thanks for flagging @Adesh-Ramchurn !